gulpfile.js

```diff
@@ -466,6 +466,8 @@ gulp.task('build', [
 ]);
 
 function nodeTest(specFiles, cb) {
+  require('./build/test/node-env-setup');
```

Comment:
this seems unrelated. Can you explain the reason for this?

Reply:
In order to test the changes and protect against regressions, the tests are run with the symbol prefix changed to` __zone_symbol_test__`. This adds the environment setup that sets the `__Zone_symbol_prefix` optin.